docs/pivoting.md

```diff
@@ -0,0 +1,142 @@
+# Pivoting/Move
```

Comment:
I'm not sure it's necessary to use both terms every time we refer to the process. Which term is preferred? Let's use that in the title, and mention the alternative once in the introduction, then use the preferred term everywhere else.

Reply:
Agree. I have changed it accordingly.